app/controllers/pro_register_controller.rb

```diff
@@ -12,7 +12,7 @@ def create
       flash["success"] = "Logged in as #{@pro.full_name}."
       redirect_to pro_dashboard_path
```

Comment:
Could try to do this in one line like this:

```ruby
redirect_to pro_dashboard_path, success: "Logged in as #{@pro.full_name}."
```

You should try for this with all of your redirects/renders. If you are using flash types that aren't the standard Rails flash types, try this: http://api.rubyonrails.org/classes/ActionController/Flash/ClassMethods.html#method-i-add_flash_types